packages/contracts/src/governance/MainVotingPlugin.sol

```diff
@@ -55,7 +57,23 @@ contract MainVotingPlugin is Addresslist, MajorityVotingBase, IEditors, IMembers
         address indexed creator,
         uint64 startDate,
         uint64 endDate,
-        string contentUri,
+        string editsContentUri,
+        address dao
+    );
+
+    /// @notice Emitted when a new flag content proposal is created.
+    /// @param proposalId Unique identifier of the proposal.
+    /// @param creator Address of the user that created the proposal.
+    /// @param startDate The timestamp when the proposal becomes active.
+    /// @param endDate The timestamp when the proposal ends.
+    /// @param flagContentUri URI pointing to the proposal's content that will be flagged.
+    /// @param dao Address of the DAO associated with the proposal.
+    event FlagContentProposalCreated(
+        uint256 indexed proposalId,
+        address indexed creator,
+        uint64 startDate,
+        uint64 endDate,
+        string flagContentUri,
```

Comment:
Would it make sense to reference the flagged proposal by it's edit/proposal id as well? Usually in the system we refer to changes by their id rather than their content hash/uri.